test/DisplayCervicalCancerMedicalHistory/cases/HistologyRefObservationMissing.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Histology Ref Observation Missing
+
+# A DiagnosticReport references an Observation but the Observation is missing
+
+data:
+-
+  resourceType: Patient
+  name: Joanne Smith
+  gender: female
+  birthDate: 1980-01-01
+  extension:
+  -
+    url: http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex
+    valueCode: F
+
+# DiagnosticReport references Observation
+-
+  resourceType: DiagnosticReport
+  code: LOINC#1234 Dummy LOINC code
+  status: final
+  effectiveDateTime: 2020-05-08T19:46:00Z
+  issued: 2020-05-10T16:42:59Z
+  result:
+  -
+    reference: Observation/1234
+
+# there is no Observation/1234 present
```

Comment:
Similar comment to one I had on another test, but I'm a little confused by the changed logic in DashboardLibrary.cql. Why would we want to be throwing an error for an irrelevant DiagnosticReport that references a missing Observation, when the DiagnosticReport is not related to cervical cancer screening?

Reply:
Our code evaluates AllCompletedDiagnosticReport because there are cases where a histology result is associated with a diagnostic report which is NOT a biopsy. This is testing that. The actual code for the DR doesn't matter here - we are not looking at the coding of the DR, but rather the histology result associated with it (if any). One method we use is by looking at Observations referenced in a DiagnosticReport. This test checks the case where the DR references an Observation which is not present, and thus is missing a result.

Reply:
My thinking of this logic was that Any DiagnosticReport referencing a Biopsy Observation should be counted as a biopsy. But Any DiagnosticReport that doesn't reference an observation should not be interpreted as a biopsy with a missing result. Yes, this theoretically _could_ be a biopsy with a missing result, but I think it is much more likely that this would be a random DiagnosticReport, ultimately causing errors to be thrown a lot more often then we would like them to.

Reply:
Discussed last Friday, when we agreed upon new logic for `HistologyDiagnosticReports`. This logic was just added today.